packages/backend/server/src/modules/workspaces/resolver.ts

```diff
@@ -82,6 +83,9 @@ export class WorkspaceType implements Partial<Workspace> {
     description: 'Members of workspace',
   })
   members!: InviteUserType[];
+
+  @Field({ description: 'Blobs size of workspace' })
+  blobsSize!: number;
```

Comment:
field resolver at L255 will add filed automatically, no need to manually adding it

Reply:
members also has a resolver, what different about both?

Reply:
oh, that must be a mistake, I will clean graphql resolvers once I have time.